BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/json_file/views/_json_raw.html.erb

```diff
@@ -0,0 +1,9 @@
+<p>
+The link below will open this JSON file directly in your browser.
+<p>
+Some modern browsers have a built-in JSON viewer.
+<p>
+
+<%= link_to "Browser-based viewer for JSON content",
```

Comment:
I personally prefer that the link opens in a new tab, and it is what it´s done for raw Text file for example. We should decide which behaviour is better.

Reply:
I didn't think of it, but it's a good idea. It should be easy to do...

Reply:
yes  `:target => "_blank"` and that's it.

Reply:
I'll do it tomorrow.